docs/getting_started/examples.rst

```diff
@@ -0,0 +1,67 @@
+
+Examples and Resources 
+========================
+
+The earlier sections of have covered the basics of using Modin to speed up your pandas workflow. Here you can find additional resource to learn about Modin. To learn more about advanced usage for Modin, please refer to :doc:`this section </advanced_usage/index>`.
+
+Basic Usage Examples
+---------------------
+
+The following notebooks demonstrate how Modin can be used for scalable data science: 
+
+- Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]
+- Using Modin with the NYC Taxi Dataset [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi.ipynb>`_] 
+- Using Modin with the NYC Taxi Dataset on a AWS cluster [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi_cloud.ipynb>`_] 
+- Modin for Machine Learning with scikit-learn [`Source <https://github.com/modin-project/modin/blob/master/examples/modin-scikit-learn-example.ipynb>`_] 
+
+Tutorials  
+----------
+
+The following tutorials cover the basic usage of Modin. `Here <https://www.youtube.com/watch?v=NglkafEmbhE>`_ is a one hour video tutorial that walks through these basic exercises.
+
+- Exercise 1: Introduction to Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/tutorial_Sources/introduction/exercise_1.ipynb>`_]
+- Exercise 2: Speed Improvements with Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/tutorial_Sources/introduction/exercise_2.ipynb>`_]
+- Exercise 3: Defaulting to pandas with Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/tutorial_Sources/introduction/exercise_3.ipynb>`_]
+
+
+The following tutorials covers more advanced features in Modin: 
+
+- Exercise 4: Experimental Features in Modin (Spreadsheet, Progress Bar) [`Source <https://github.com/modin-project/modin/blob/721d2d2fe2a46d3b7442e1ab2c7b70ad3bff0712/examples/tutorial/tutorial_Sources/introduction/exercise_4.ipynb>`_]
+- Exercise 5: Setting up Modin in a Cluster Environment [`Source <https://github.com/modin-project/modin/blob/721d2d2fe2a46d3b7442e1ab2c7b70ad3bff0712/examples/tutorial/tutorial_Sources/cluster/exercise_5.ipynb>`_]
+- Exercise 6: Running Modin in a Cluster Environment [`Source <https://github.com/modin-project/modin/blob/721d2d2fe2a46d3b7442e1ab2c7b70ad3bff0712/examples/tutorial/tutorial_Sources/cluster/exercise_6.ipynb>`_]
+
+Talks & Podcasts
```

Comment:
There are two more talks that might be added.

https://www.youtube.com/watch?v=oo_lxUjsFTM&t=1s
https://www.youtube.com/watch?v=cOM82kHRwkM&t=6568s

Reply:
but we have to add a note, that they're in Russian, not in English

Reply:
added talk links